### PR TITLE
API review 7: Simple stuff--renames, namespace changes, etc.

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

**Comment by ErikEJ** [comment]
Do you plan to include this in beta 6?

**Comment by ajcvickers** [comment]
@ErikEJ Its not currently planned to be in beta 6.

File: test/EntityFramework7.Relational.Tests/Metadata/RelationalBuilderExtensionsTest.cs [hunk 1]
```diff
@@ -358,10 +358,10 @@ public void Can_create_named_sequence_with_specific_facets_with_convention_build
 
             modelBuilder
                 .Sequence("Snook")
-                .IncrementBy(11)
-                .Start(1729)
-                .Min(111)
-                .Max(2222)
+                .IncrementsBy(11)
+                .HasStart(1729)
```
Comment (bricelam) [inline test/EntityFramework7.Relational.Tests/Metadata/RelationalBuilderExtensionsTest.cs]: This name feels a little strange. Maybe`StartsAt`...

**Comment by bricelam** [comment]
:shipit:

File: test/EntityFramework7.Relational.Tests/Metadata/RelationalBuilderExtensionsTest.cs (hunk 1 above)
Comment (ajcvickers) [inline test/EntityFramework7.Relational.Tests/Metadata/RelationalBuilderExtensionsTest.cs]: Agreed. We has a blanket comment to prefix Has, but I agree StartsAt is better. Will do.

**Comment by ajcvickers** [comment]
Merged

**Pull request closed without merging**